source/help/getting-started/organizing-conversations.rst

```diff
@@ -154,7 +171,11 @@ When configured by the System Admin, users can enable the features in **Account
 
 **Collapsible custom categories**: Create custom categories in the sidebar to group channels together for easier navigation (e.g. “Design” or “Marketing”). Collapse categories to display only unread channels and reduce unnecessary scrolling.
 
-**Drag and drop channels and categories**: Drag channels between or within categories, or drag to reorder entire categories to prioritize important conversations.
+**Drag and drop channels and categories**: Drag channels between or within custom categories, or drag to reorder entire categories to prioritize important conversations.
+
+**Rename custom categories**: Customize name of categories.
+
+**Mute and unmute custom categories**: Mute and unmute all channels in a custom category.
```

Comment:
You can also mute a category, then choose to unmute select channels, or vise versa

Reply:
Thanks!